doc/installation.md

```diff
@@ -15,12 +15,12 @@ fifty_deg_sylius_cache_plugin:
     resource: "@FiftyDegSyliusCachePlugin/Resources/config/routes.yaml"
 ```
 
-4. Import Webpack settings from `vendors/fifty-deg/sylius-cache-plugin/webpack.config.js` in your `webpack.config.js` file. Please, note that `fiftyDegSyliusCachePluginAdmin` is required while is optionalm see <a href="./usage.md" target="_blank">Usage</a> docs for more informations.
+4. Import Webpack settings from `vendors/fifty-deg/sylius-cache-plugin/webpack.config.js` in your `webpack.config.js` file.
```

Comment:
Hi @MarcoPistorelloAtFiftyDeg , please, update the description in order to match the code snippet, see `'./vendor/fifty-deg/sylius-cache-plugin/webpack.config'`. Thanks!

Reply:
Hi @FilippoTascaAtFiftyDeg, installation.md has already the correct import, maybe there was a commit executed later than this one shown above.

Reply:
I'm talking about the description at point `4.` @MarcoPistorelloAtFiftyDeg: looking at the diff, the description is 
```
4. Import Webpack settings from `vendors/fifty-deg/sylius-cache-plugin/webpack.config.js` in your `webpack.config.js` file.
```

Thanks!